importers/src/main/groovy/whelk/importer/ImporterMain.groovy

```diff
@@ -56,42 +56,6 @@ class ImporterMain {
         new CardRefresher(whelk).refresh(collection)
     }
 
-    @Command(args='[COLLECTION] [no-embellish|no-cache]')
-    void reindexToStdout(String collection=null, String directive=null) {
-        if (directive == null && collection?.indexOf('-') > -1) {
-            directive = collection
-            collection = null
-        }
-        boolean useCache = directive != 'no-cache'
-        boolean doEmbellish = directive != 'no-embellish'
-
-        Whelk whelk = Whelk.createLoadedCoreWhelk(props, useCache)
-
-        println "Creating whelk with dummy ElasticSearch"
-        println "- collection: $collection"
-        println "- directive: $directive"
-        println "- useCache: $useCache"
-        println "- doEmbellish: $doEmbellish"
-
-        whelk.elastic = new ElasticSearch("", "", "") {
-            Tuple2<Integer, String> performRequest(String method,
```

Comment:
This method doesn't exist...

Reply:
We've either deleted that in a cleanup, or I threw this together with a local change. :stuck_out_tongue_closed_eyes: It was for performance testing gone by, so no need to keep it around.